Change to domain name mapping

KaneBrennan132 opened the pull request:
Changed the domain-name mapping as DOMAINNAME(domainid) is for the Qradar tenancy domains.

User documentation changes are planned.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/opencybersecurityalliance/stix-shifter?pullRequest=1342) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/opencybersecurityalliance/stix-shifter?pullRequest=1342) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/opencybersecurityalliance/stix-shifter?pullRequest=1342) it.</sub>

delliott90 commented on stix_shifter_modules/qradar/stix_translation/json/stix_2_1/to_stix_map.json:
```diff
@@ -465,7 +465,9 @@
-    },
+    }
+  ],
+  "qradardomain": [
```
You should remove the entry that's currently under `dnsdomainname` a few lines down. We don't want to write duplicated data in multiple properties. In fact, if dnsdomainname is truly a domain, it shouldn't be writing the value to the url object.

delliott90 reviewed (changes requested):
Any changes to the `to-stix` mapping files should be reflected in the `from-stix` files as well.

KaneBrennan132 commented:
Changed dnsdomainname entry to only write to domain-name
Changed alias used by DNS request domain in event search to use the alias dnsdomainname, removed domainname entry as it is no longer required. 
Updated Unit Tests to reflect changes
Updated Docs to reflect changes
